nextflow_schema.json

```
@@ -18,6 +18,7 @@
                 },
                 "skip_tools": {
                     "type": "string",
+                    "pattern": "^(fastp|fastqc|kraken|multiqc|checkqc|falco|md5sum|samshee)(,(fastp|fastqc|kraken|multiqc|checkqc|falco|md5sum|samshee)?,?)*(?<!,)$",
```

There seems to be some duplication in this pattern, I think we can do better.
This should do the trick:
```suggestion
                    "pattern": "^((fastp|fastqc|kraken|multiqc|checkqc|falco|md5sum|samshee)?,?)*(?<!,)$",
```

cf https://github.com/nf-core/sarek/blob/master/nextflow_schema.json#L122

---

Thank you for your suggestion, I added this!